services/ui-src/src/verbiage/pages/home.ts

```diff
@@ -23,36 +23,30 @@ export default {
         "https://www.govinfo.gov/content/pkg/PLAW-109publ171/pdf/PLAW-109publ171.pdf",
       postLinkText:
         ' as "increasing the use of home and community-based, rather than institutional, long-term care services."',
-      downloadText: "",
+      downloadText: "User Guide and Help File",
       link: {
         text: "Enter Work Plan online",
         route: "wp",
       },
       accordion: {
         buttonLabel: "When is the MFP Work Plan due?",
-        text: "The MFP Work Plan will be created and submitted initially and then amended semi-annually. CMS will review and approve all MFP Work Plans and proposed changes.The MFP Work Plan deadlines are May 1st for Reporting Period 1 and November 1st for Reporting Period 2. Updates to existing state or territory-specific initiatives or the addition of new initiatives can occur during either of these periods, however, changes to transition benchmark projections should be made between July - November 1st and align with program budget requests. CMS approved changes will be integrated into the MFP recipient’s Semi-Annual Progress Report for the following reporting period.",
+        text: "The MFP Work Plan will be created and submitted initially and then amended semi-annually. CMS will review and approve all MFP Work Plans and proposed changes.<br/><br/>The MFP Work Plan deadlines are May 1st for inclusion in January through June reporting (Reporting Period 1) and November 1st for inclusion in July through December reporting (Reporting Period 2). Updates to existing state or territory-specific initiatives or the addition of new initiatives can occur during either of these periods, however, changes to transition benchmark projections should be made between July - November 1st to align with program budget requests.<br/><br/>CMS approved changes will be integrated into the MFP recipient's Semi-Annual Progress Report for the following reporting period.",
```

Comment:
Superduper pedantic question because I'm an old geezer:  just one space after periods?

Reply:
I don't think we have a specific standard for that, but I'm happy to take suggestions! Should we be adding more spaces after a period?

Reply:
Lol I think we've been doing one space after periods everywhere

Reply:
If that's the standard!  I'm just an old fudd that was taught to put two after breaks like periods, semicolons, and colons.

Reply:
https://silicon-valley.fandom.com/wiki/Tabs_vs._spaces